astropy/tests/runner.py

```diff
@@ -181,6 +197,16 @@ def run_tests(self, **kwargs):
 
         args = self._generate_args(**kwargs)
 
+        if 'plugins' not in self.keywords or self.keywords['plugins'] is None:
+            self.keywords['plugins'] = []
+
+        # Make plugins available to test runner without registering them
+        self.keywords['plugins'].extend([
+            'astropy.tests.plugins.display',
+            'astropy.tests.plugins.repeat',
+            'astropy.tests.plugins.config'
+        ])
```

Comment:
@astrofrog see here for how we make sure `astropy.test()` uses these plugins without registering them in `setup.py`.